src/components/upload-promotion/UploadPromotionForm.less

```diff
@@ -0,0 +1,38 @@
+.upload-promotion-form {
```

Comment:
So less allows you to define the scope of the ant properties to just the upload-promotion-form right? This is pretty useful

Reply:
Kind of! The scope of the ant rules defined in this file is already restricted to the files that import this file. But here Less allows us to do this "nesting" of CSS rules which just makes it easier to read for developers since it's in a hierarchy kind of format 😛